src/components/Statistics/FeedbackItem.vue

```diff
@@ -28,8 +28,12 @@
               <DateAsWords :date="createdAt" />
             </span>
           </small>
-          <div class="comment">{{ comment }}</div>
-          <div>
+          <div
+            v-if="comment"
+            class="comment">
```

Comment:
Ah, I just saw that the comment input is a textarea. So it would make sense to make it `multiline` too.

Reply:
Yeah, makes sense. Was also just wondering about that. -> Changed it :white_check_mark: